src/triton_cli/constants.py

```diff
@@ -44,3 +44,6 @@
 DEFAULT_HF_CACHE: Path = Path.home() / ".cache" / "huggingface"
 HF_CACHE: Path = Path(os.environ.get("TRANSFORMERS_CACHE", DEFAULT_HF_CACHE))
 SUPPORTED_BACKENDS: set = {"vllm", "tensorrtllm"}
+
+# Parser
+PASSTHROUGH_SUBCOMMANDS = ["profile", "optimize"]
```

Comment:
Moving to thread :thread:

Reply:
@dyastremsky :

> I think we're 100% going for a passthrough approach here. This makes the Triton CLI extensible and maintains all logic unique to the tools in their own repositories. If we start moving over logic, then responsibilities are starting to overlap and we could be duplicating code. If there is a required arg for profile and we think it should set a default, Model Analyzer is the right place to fix that.

Reply:
Agreed that passthrough as much as possible will be the simplest to maintain, and fixing ergonomics in the downstream tools being invoked would be preferred. There may be some cases to fork logic in CLI, but should be few and far between if we can help it.